### PR TITLE
Update supported orientation mask examples

### PR DESCRIPTION
**Pull request by Bucimis**

### CODE REVIEW

File: _docs/_developer_guide/platform_integration_guides/ios/in-app_messaging/customization.md [hunk 1]
```diff
@@ -130,8 +131,10 @@ To set a fixed orientation for all in-app messages, you can set the `supportedOr
 
 ```swift
 // Set fixed in-app message orientation to portrait
-Appboy.sharedInstance()?.inAppMessageController.supportedOrientationMasks = portrait;
-// Use landscape to display in-app messages in landscape
+if let controller = Appboy.sharedInstance()?.inAppMessageController.inAppMessageUIController as? ABKInAppMessageUIController {
+  controller.supportedOrientationMask = .portrait
+}
+// Use .landscape to display in-app messages in landscape
```
Comment (hokstuff) [inline _docs/_developer_guide/platform_integration_guides/ios/in-app_messaging/customization.md]: We should move this comment up into the if statement block

File: _docs/_developer_guide/platform_integration_guides/ios/in-app_messaging/customization.md [hunk 2]
```diff
@@ -121,7 +121,8 @@ To set a fixed orientation for all in-app messages, you can set the `supportedOr
 
 ```objc
 // Set fixed in-app message orientation to portrait.
-[Appboy sharedInstance].inAppMessageController.inAppMessageUIController.supportedOrientationMasks = UIInterfaceOrientationMaskPortrait;
+id inAppMessageUIController = [Appboy sharedInstance].inAppMessageController.inAppMessageUIController;
```
Comment (hokstuff) [inline _docs/_developer_guide/platform_integration_guides/ios/in-app_messaging/customization.md]: Optional: we can update this to `id<ABKInAppMessageUIControlling> inAppMessageUIController =` to be explicit with what's going on

File: _docs/_developer_guide/platform_integration_guides/ios/in-app_messaging/customization.md (hunk 1 above)
Comment (Bucimis) [reply _docs/_developer_guide/platform_integration_guides/ios/in-app_messaging/customization.md]: updated for both

File: _docs/_developer_guide/platform_integration_guides/ios/in-app_messaging/customization.md (hunk 2 above)
Comment (Bucimis) [reply _docs/_developer_guide/platform_integration_guides/ios/in-app_messaging/customization.md]: nice, fixed

**Comment by Bucimis** [comment]
@EmilyNecciai please merge

**Pull request merged**